SecretSanta/src/SecretSanta.Api/Controllers/UsersController.cs

```diff
@@ -19,22 +21,48 @@ public UsersController(IUserRepository repository)
         }
 
         [HttpGet]
-        public IEnumerable<User> Get()
+        [ProducesResponseType(typeof(IEnumerable<User>), StatusCodes.Status200OK)]
+        //public IEnumerable<User> Get()
+        public IEnumerable<FullUser> Get()
```

Comment:
There is a disjoint here between the return type (`IEnumerable<FullUser>`) and the declared returned type (`IEnumerable<User>`).
```suggestion
        [ProducesResponseType(typeof(IEnumerable<FullUser>), StatusCodes.Status200OK)]
        //public IEnumerable<User> Get()
        public IEnumerable<FullUser> Get()
```